test/cross-browser/reference.html

```diff
@@ -22,36 +20,51 @@
 
     <!-- traces -->
     <script>
-        var childWin;
+        var clientLogger;
+
+        function createLocalStorageLogger() {
+            return {
+                debug: function() {
+                    localStorage.setItem(
```

Comment:
This is a blanket comment. The majority of our implemented code, accesses localStorage using bracket notation rather than `setItem` and `getItem`. It appears both have near identical behavior: http://stackoverflow.com/questions/12632463/is-localstorage-getitemitem-better-than-localstorage-item-or-localstoragei?answertab=active#tab-top

If it were me I would change to use bracket notation (I prefer consistency), but ultimately your call.